source/administration/batch-framework.rst

```diff
@@ -84,9 +87,14 @@ Job Types
 Replicate
 ~~~~~~~~~
 
-Use the ``replicate`` job type to create a batch job that replicates objects from the local MinIO deployment to another MinIO location.
+Use the ``replicate`` job type to create a batch job that replicates objects from one MinIO deployment to another MinIO location.
+At least one of the deployment locations, either the source or the target, must be ``local``.
 The definition file can limit the replication by bucket, prefix, and/or filters to only replicate certain objects.
 
+.. versionchanged:: MinIO RELEASE.2023-04-07T05-28-58Z
+
+   You can replicate from a remote MinIO deployment to the local deployment that runs the batch job.
+
 For example, you can use a batch job to perform a one-time replication sync of objects from ``minio-alpha/invoices/`` to ``minio-baker/invoices``.
```

Comment:
Since before we're talking about "local" and "remote", maybe label one of these as the "local" and one as the "remote" and note that it works in both directions.

spitballing

"For example, you can create a batch job that performs a one-time "push" replication sync on the "local" ``minio-alpha/invoices`` deployment from the remote ``minio-baker/invoices`` resource.
The batch job could also do the reverse, "pulling" from the remote resource into the local."

Lots of quote-y things so edit at will.